qemu/tests/qemu_guest_agent.py

```diff
@@ -894,6 +894,28 @@ def gagent_check_thaw_unfrozen(self, test, params, env):
             test.fail("The return value of thawing an unfrozen fs is %s,"
                       "it should be zero" % ret)
 
+    @error_context.context_aware
+    def gagent_check_freeze_frozen(self, test, params, env):
+        """
+        Freeze the frozen fs
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        self.gagent.fsfreeze()
+        error_context.context("Freeze the frozen FS", logging.info)
+        try:
+            self.gagent.fsfreeze(check_status=False)
+        except guest_agent.VAgentCmdError, e:
+            expected = ("The command guest-fsfreeze-freeze has been disabled "
+                        "for this instance")
```

Comment:
Is there any difference between rhel6 and rhel7 for this expected result ?

Reply:
After confirm with xuhan, no need to diff rhel6 if no plan to run it on rhel6 host.
So no issues now.

Reply:
The guest agent feature will not be tested on host 6